Rest/Helper/FieldTypeValueHelper.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/**
+ * This file is part of the EzSystemsRecommendationBundle package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Rest\Helper;
```

Comment:
OK, it's not an issue but...

First of all, you shouldn't use `helper` name here (see https://github.com/ezsystems/EzSystemsRecommendationBundle/pull/10#commitcomment-10216846).

Secondly, your helpers are doing to much. It's only mi opinion but I think it would be better to split it into smaller services.

Besides in the `FieldValueHelper` you're injecting `FieldTypeValueHelper` service - isn't that too much? ;)